project/Dependencies.scala

```diff
@@ -8,6 +8,7 @@ object Dependencies {
   lazy val typesafe = "com.typesafe" % "config" % "1.4.0"
   lazy val scalaTags = "com.lihaoyi" %% "scalatags" % "0.8.2"
   lazy val awsUtils =  "uk.gov.nationalarchives.aws.utils" %% "tdr-aws-utils" % "0.1.4"
+  lazy val loggerSlf4j = "org.typelevel" %% "log4cats-slf4j" % "1.2.0-RC2"
```

Comment:
Is there a stable release we can use rather than a release candidate?

Reply:
No, there's only a choice of two https://mvnrepository.com/artifact/org.typelevel/log4cats-slf4j 
There is an older `libraryDependencies += "io.chrisdavenport" %% "log4cats-slf4j" % "1.1.1"` but that hasn't been updated in a while and the github [README](https://github.com/typelevel/log4cats) says to use the typelevel organisation.

Reply:
OK. If we see any issues with the RC, I think we should downgrade to 1.1.1. Chris Davenport seems to be the main developer, so that old version looks like it is genuinely the most recent stable release.